scripts/setup.sh

```diff
@@ -77,7 +77,7 @@ clients:
     # if the user provides ANY of the listed credentials, the login is successful
     requirements:
       - cTypeHash: "0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac"
-        trustedAttesters: ["did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY"]
+        trustedAttesters: ["did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY","did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare"]
```

Comment:
So these are new the DIDs of SKYC on both Peregrine and Spiritnet? Or what?

Reply:
why are these hard-coded? And can we please add comments explaining those values?

Reply:
I am not a big fan of adding spiritnet skyc's DID there. You can get a credential from peregrine and login. Is a security threat.

Reply:
yes `w3n:attester` on peregrine and `w3n:socialkyc` on spiritnet are listed as trusted attesters of an email credential. I think that info is publicly available yet added comments as suggested

Reply:
Yes but I agree with Raphael that these should not be hardcoded, right? Why would you trust `w3n:attester` on Peregrine if you have deployed your system on Spiritnet? Making these configurable as well would make sense.

Reply:
I made the [trusted attester configurable.](https://github.com/KILTprotocol/opendid/commit/5cf62fa6e77067a8273566deecc8a255f35fe812) I left the ctypehash as before though

Reply:
I would say the CType hash should also be parametrised. I'll resolve this comment for the sake of unblocking you, but I opened a ticket for it: https://github.com/KILTprotocol/ticket/issues/3138. Although I am not sure, if we parametrise CType, the video becomes outdated. Videos are very dangerous in this regard as it's really hard to know when they become outdated. Do we have a strategy about that?